app/views/cookies/new.html.haml

```diff
@@ -0,0 +1,27 @@
+= govuk_page_title(t('cookie_settings.page_title'))
+
+%p.govuk-body= t('cookie_settings.section_intro.paragraph_one')
+
+%p.govuk-body= t('cookie_settings.section_intro.paragraph_two')
+
+%h2.govuk-heading-l= t('cookie_settings.section_necessary.heading')
+
+%p.govuk-body= t('cookie_settings.section_necessary.paragraph_one')
+
+%ul.govuk-list.govuk-list--bullet
+  %li= t('cookie_settings.section_necessary.list_item_one')
+  %li= t('cookie_settings.section_necessary.list_item_two')
+
+%p.govuk-body= t('cookie_settings.section_necessary.paragraph_two')
+
+%p.govuk-body
+  = link_to t('cookie_settings.section_necessary.link_to_cookie_page'), cookies_path, class: 'govuk-link'
+
+.app-no-js-only
+  %h2.govuk-heading-l= t('cookie_settings.section_settings.heading_no_js')
+
+  %p.govuk-body= t('cookie_settings.section_settings.paragraph_one_no_js')
+
+  %p.govuk-body= t('cookie_settings.section_settings.paragraph_two_no_js')
```

Comment:
Is this code block still required?

Reply:
I think so. We still should not be asking the question to people without JS.

Reply:
As it stands, the banner will be displayed irrespective of whether JS enabled so this should also not be shown in that case

Reply:
We've implemented a slightly different banner and form for where JS is not enabled, so the form will not be rendered if the user has disabled JS